application-forum-ui/src/main/resources/ForumCode/Macros.xml

```diff
@@ -535,6 +535,15 @@ $value.replaceFirst($regextool.quote('{{html clean="false" wiki="false"}}'), '')
 ##
 ##
 #macro(getOrdredTopics $forum $start $nb $sortField $varName)
+  #set ($supportsNS = $services.extension.core.getCoreExtension('org.xwiki.platform:xwiki-platform-model').id.version.compareTo($services.extension.parseVersion('7.2')) &gt;= 0)
+  #if ($supportsNS)
+  ## define some variables to be used with Nested Spaces
+    #set($hierarchyCondition = "doc.space=:forum and ")
+    #set($hierarchyBinding = $doc.space)
```

Comment:
What if the forum is not the space of the document. Can it happen? I'm not sure I understand this line.